modules/@apostrophecms/modal/ui/apos/mixins/AposDocsManagerMixin.js

```diff
@@ -88,6 +88,14 @@ export default {
     }
   },
   methods: {
+    // It would have been nice for this to be computed, however
+    // AposMediaManagerDisplay does not re-render when it is
+    // a computed prop rather than a method call in the template.
+    maxReached() {
+      // Reaching max and exceeding it are different things
+      const result = this.relationshipField.max && this.checked.length >= this.relationshipField.max;
```

Comment:
This is odd as a method, I agree. If the issue is the display view not updating, we could have a watch listener in there? But I'm not too offended by this.